homework_03/03_atanas_atanasov/circles.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+
+class point:
+	xcoord = 0
+	ycoord = 0
+
+	def __init__(self, x, y):
+		self.xcoord = x
+		self.ycoord = y
+
+	def toString(self):
+		print("x: ", self.xcoord, "\ny: ", self.ycoord)
+
+class circleStates(Enum):
+	NO_COMMON_POINTS = 0
+	TOUCHING = 1
+	INTERSECTING = 2
+	SAME = 3		
+
+class circle:
+	center = 0
+	radius = 0
+
```

Comment:
There is no need to define the members at the start of the class

Reply:
Vladi is right, you don't have to define it at the start of the class.